core/templates/dev/head/pages/exploration_editor/editor_navigation_directive.html

```diff
@@ -81,3 +81,28 @@
     {% endif %}
   </ul>
 </script>
+<style type="text/css">
+.popover{
+  font-size: 13px;
+  max-width: 300px;
+  text-align: center;
+  z-index: -1;
+  color: white;
+  background-color: black;
+}
```

Comment:
@kevinlee12  @shivanXI  I would request to keep this PR stagnant as there are basic media only changes happening in my PR due to _**5 second popup disappear rule**_ #3093. As you might see the [changes here](https://github.com/oppia/oppia/pull/3267/files) 
isLargeScreen is been modified. I will report further changes as #3093 PR progresses